recipes/packages.rb

```diff
@@ -20,4 +20,4 @@
   package p
 end
 
-include_recipe 'php::package'
```

Comment:
Why remove the package recipe?

Reply:
The TL;DR is that it's for the same reason I did last time: not doing so causes conflicts between the php cookbook's packages and the ones we specify w/ osl-php's attributes. This is not apparent in testing just `osl-php`, but it is if you try it with `proj-phpbb`.

In the version `1.2.6` of the php cookbook, `php::package` installs the packages in `node['php']['packages']` and creates the `php.ini` file from a template.

In version `1.7.2` of the php cookbook, `php::package` also installs the packages in `node['php']['packages']`, then includes `php::ini` which creates the `php.ini` file from a template.

We use `osl-php` to install php by setting the packages we want in `node['osl-php']['packages']`, as that's what the `osl-php::packages` recipe does. However, the recipe fails to converge if we set our wrapper's packages attribute but fail to clear the php cookbook's attribute because the php packages conflict (unless they're exactly the same).